ios/Wrappers/BinaryDataWrapper.swift

```diff
@@ -0,0 +1,24 @@
+//
+//  BinaryDataWrapper.swift
+//
+
+
+import Foundation
+import MessagePacker
+
+enum BinaryDataWrapperError: Swift.Error {
+	case encodeError(String)
+}
+
+protocol BinaryDataWrapper: Codable {
+	associatedtype T
+	static func wrap(model: T) throws -> Self
+}
+
+extension BinaryDataWrapper {
```

Comment:
Not an issue for this PR, but I'm curious how this performs vs the JSON wrapper we're using for the other types. If it's faster we might want to move the other wrappers to use this as well. Just an idle thought, no action needed here.